.changeset/few-mice-notice.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-frontend/ui-kit': patch
```

Comment:
```suggestion
'@commercetools-uikit/table': patch
```

Only the table package got updated

Reply:
Sorry, updated. (also 'visual-testing-app')